chart/openfaas/values.yaml

```diff
@@ -38,6 +38,36 @@ gateway:
       memory: "120Mi"
       cpu: "50m"
 
+basicAuthPlugin:
+  image: openfaas/basic-auth-plugin:0.17.0
+  replicas: 1
+  resources:
+    requests:
+      memory: "50Mi"
+      cpu: "20m"
+
+oauth2Plugin:
+  license: "xyz"
+  enabled: true
```

Comment:
Should this be try by default?

Reply:
For the time being, it might be easier for testing the PR. Needs to be off by default for merge.